common/culture/traditions/00_realm_traditions.txt

```diff
@@ -2674,8 +2702,19 @@ tradition_hidden_cities = {
 
 	is_shown = {
 		OR = {
+			# Warcraft
 			like_heritage_akan_trigger = yes
 			like_heritage_yoruba_trigger = yes
+			this = culture:zandalari
+			this = culture:gurubashi
+			this = culture:goblin
+			this = culture:jungle_hozen
```

Comment:
Yeah, the tradition sounds nice for trolls, but why they don't have it from the start?

Reply:
I balanced between a lore/gameplay balance in selecting the traditions. You can check the alternatives I listed for each culture in the spreadsheet. Which tradition would you propose replacing?

Reply:
Zandalari - tradition_hereditary_hierarchy
Gurubashi - tradition_medicinal_plants

I think this tradition is a core one since it allows to have nice bonuses and jungles and players will prefer to have them anyway.